dracut/modules.d/90kiwi-live/kiwi-generator.sh

```diff
@@ -36,6 +36,10 @@ ROOTFLAGS="$(getarg rootflags)"
     _dev=LiveOS_rootfs
 } > "$GENERATOR_DIR"/sysroot.mount
 
+mkdir -p "$GENERATOR_DIR"/initrd-root-fs.target.requires
+ln -s "$GENERATOR_DIR"/sysroot.mount \
+    "$GENERATOR_DIR"/initrd-root-fs.target.requires/sysroot.mount
```

Comment:
What if sysroot.mount already exists ?

Reply:
mmm it should not happen `fstab-generator` does not create it in `generator.early` as we do, but you are right, having a check would be safer. I'll include it